src/textures.c

```diff
@@ -523,7 +541,7 @@ static int texPngLoadAll(GSTEXTURE *texture, const char *filePath, int texId)
     png_set_filler(pngPtr, 0xff, PNG_FILLER_AFTER);
     png_read_update_info(pngPtr, infoPtr);
 
-    void (*texPngReadPixels)(GSTEXTURE * texture, png_bytep * rowPointers, size_t size);
```

Comment:
Unneeded formatting change?

Reply:
not sure, but previously it was not clang-compatible. Pointers everywhere doesnt have 2 spaces around them, and i made checks on localy installed clang

Reply:
maybe clang-fomrat in this action has a bug?

Reply:
The clang-format version should be 12. Different clang format version will give different output